Guide.docc/CommonProblems.md

```diff
@@ -92,6 +92,16 @@ disallow mutation, guaranteeing safe read-only access.
 let supportedStyleCount = 42
 ```
 
+A constant value can also be expressed with a computed property.
```

Comment:
I would not call it a constant, maybe just global
```suggestion
A global value can also be expressed with a computed property.
```

Reply:
I get what you mean that it's "the same value always" but maybe express this in text rather than calling it a constant